nping/output.cc

```diff
@@ -244,10 +244,11 @@ int nping_print(int level, const char *str, ...){
   }
 
   /* If supplied level is more than current level, do nothing */
-  if( level>=QT_4 && level<=VB_4 && level>current_vb_level )
-    return OP_SUCCESS;
-  if( level>=DBG_0 && level<=DBG_9 && level>current_dbg_level )
-    return OP_SUCCESS;
+  if(( level>=QT_4 && level<=VB_4 && level>current_vb_level ) ||
+    ( level>=DBG_0 && level<=DBG_9 && level>current_dbg_level )) {
+      va_end (list);
+      return OP_SUCCESS;
+    }
```

Comment:
I am not sure why your analyzer has not picked it up but the same issue is also present in almost identical routines `nping_fatal` and `nping_warning` earlier in the same source file.

Reply:
@nnposter it did actually. There're a lot of issues like that have been reported I prefer not to push huge patch which fixes all of them to not make it difficult to review/debug

Reply:
Could this be changed to simply move the `va_start` below these `return` statements? Even move both `va_start` and `va_end` inside the conditional with `vfprintf`?